### ISSUE TITLE
Copy AzCopy command to clipboard, but in bash format instead of pwsh

### ISSUE THREAD

**Issue by Balfa** [microsoft/AzureStorageExplorer#7177]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Problem

I have to manipulate the AzCopy command that was copied to my clipboard to be compatible with bash, because it always uses powershell formats for variables

### Desired Solution

Maybe a setting can toggle preferred format between bash and powershell when copying azcopy command to clipboard?

### Alternatives and Workarounds

I paste it into text editor and change, for example `$env:AZCOPY_CRED_TYPE = "Anonymous";` to `AZCOPY_CRED_TYPE="Anonymous";`

**Comment by MRayermannMSFT** [comment]
@Balfa what OS are you using?

**Comment by Balfa** [comment]
As, sorry. I should have mentioned this. I'm using Windows 10 (19045.3208), but mots of my command line work is done in git bash in windows terminal. That's why I'd like an option for git bash

**Comment by Balfa** [comment]
> @Balfa what OS are you using?

As, sorry. I should have mentioned this. I'm using Windows 10 (19045.3208), but mots of my command line work is done in git bash in windows terminal. That's why I'd like an option for git bash

**Comment by MRayermannMSFT** [comment]
- Platform Dependent
- PowerShell
- Bash

**Comment by MRayermannMSFT** [comment]
@Balfa just to check, is this ok?
```
export AZCOPY_BUFFER_GB=4;
export AZCOPY_CONCURRENT_FILES=32;
./azcopy.exe copy "https://REDACTED.blob.core.windows.net/dir/blob?sv=2019-12-12&ss=btqf&srt=sco&st=2020-08-19T20%3A09%3A50Z&se=2028-07-02T20%3A09%3A00Z&sp=rwdxlacup&sig=REDACTED" "C:\rREDACTED\File1699391437268734810.txt";
unset AZCOPY_BUFFER_GB;
unset AZCOPY_CONCURRENT_FILES;
```

**Comment by Balfa** [comment]
@MRayermannMSFT  that looks bang-on! Thanks 🙂

**Comment by MRayermannMSFT** [comment]
@Balfa one more input request from you. Do you think this title and description make sense?
![image](https://github.com/microsoft/AzureStorageExplorer/assets/21112697/783514ea-bcd8-42ae-b683-ee08cf8df8f8)

**Comment by Balfa** [comment]
@MRayermannMSFT  perfect, thanks!

**Comment by MRayermannMSFT** [comment]
Merged into main for inclusion in 1.33.
